tsconfig.json

```diff
@@ -1,13 +1,15 @@
 {
   "include": [
-    "src/**/*"
+    "src/**/*",
+    "tests/**/*"
   ],
   "compilerOptions": {
     "baseUrl": ".",
     "outDir": "dist",
-    "rootDir": "src",
+    "rootDir": ".",
```

Comment:
so that the test directory will be properly type-checked. It is also common to put tests in the source

Reply:
I think I'll need to move the tests into the src directory to avoid making this a big refactor. Basically specifying "." as the rootDir ends up changing all the `dist/` paths